content/articles/regulated-tlds.markdown

```diff
@@ -0,0 +1,36 @@
+
+---
+title: Regulated Top Level Domains
+excerpt: Information about regulated and highly-regulated TLDs.
+categories:
+- Domains
+---
+
+# Regulated and highly-regulated Top Level Domains
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+Certain top level domains (TLDs) my be subject to the registrant meeting specific requirements or regulations. This article describes basic requirements for regulated and highly-regulated TLDs and provides a list of regulated and highly-regulated TLDs.
+
+## Regulated TLDs
+
+For domain name registration in any "Regulated TLD", registrant must acknowledge and agree that the registration is subject to the following additional requirements: (a) comply with all applicable laws, including those that relate to privacy, data collection, consumer protection (including in relation to misleading and deceptive conduct), fair lending, debt collection, organic farming, disclosure of data, and financial disclosures; and (b) if the registrant collects and maintains sensitive health and financial data, they must implement reasonable and appropriate security measures commensurate with the offering of those services, as defined by applicable law.
+
+## Regulated TLD List
+
+.games, .juegos, .school, .schule, .toys, .eco, .care, .diet, .fitness, .health, .clinic, .dental, .healthcare, .capital, .cash, .broker, .claims, .exchange, .finance, .financial, .fund, .investments, .lease, .loans, .market, .money, .trading, .credit, .insure, .tax, .mortgage, .degree, .mba, .audio, .book, .broadway, .film, .movie, .music, .software, .fashion, .video, .app, .art, .band, .cloud, .data, .digital, .fan, .free, .gratis, .discount, .sale, .media, .news, .online, .pictures, .radio, .show, .theater, .tours, .accountants, .architect, .associates, .broker, .legal, .realty, .vet, .engineering, .law, .limited, .show; .theater; .town, .city, .reise, and .reisen
+
+## Highly Regulated TLDs
+
+In addition to the requirements for Regulated TLDs, domain name registration in any "Highly-Regulated TLD" is subject to the following requirements: (a) registrant will provide administrative contact information, which must be kept up‐to‐date, for the notification of complaints or reports of registration abuse, as well as the contact details of the relevant regulatory or industry self‐regulatory bodies in their main place of business; (b) registrants represent that they possess any necessary authorizations, charters, licenses and/or other related credentials for participation in the sector associated with such Highly‐regulated TLD; and (c) registrants will report any material changes to the validity of registrant authorizations, charters, licenses and/or other related credentials for participation in the sector associated with the Highly‐regulated TLD to ensure you continue to conform to the appropriate regulations and licensing requirements and generally conduct the registrant's activities in the interests of the consumers they serve.
```

Comment:
I feel like it would make sense to turn this (and the section above) into an 
a) 
b)
c) 
style list so it's a bit easier to read. Thoughts?

Reply:
Done.